.github/workflows/brew.yml

```diff
@@ -0,0 +1,20 @@
+name: brew
+
+on:
+  release:
+    types: published
+
+jobs:
+  homebrew:
+    name: Bump homebrew-core formula
+    runs-on: ubuntu-latest
+    if: ${{ needs.release-please.outputs.release_created }}
```

Comment:
I think this is probably not needed anymore after the on-release trigger

Reply:
yeah, that is right!